apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -224,7 +225,7 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
   };
 
   return (
-    <View style={commonTestStyles.flex}>
+    <View accessible={!enableSinglePaneView} testID={ROOT_VIEW} style={commonTestStyles.flex}>
```

Comment:
Shouldn't this view always be accessible if we're setting a testID on it and we want the testID to be discoverable on both mobile and non-mobile platforms?

Reply:
From what I recall (at least on win32), testID is only set if the component is accessible.

Reply:
When making this view accessible, this apparently breaks querying for elements on the iOS side, which is why the CI was continuously failing for this PR. Not entirely sure why.

Reply:
@Saadnajmi Any input here? Any idea why setting this View as accessible would break iOS testing?

@lawrencewin What is the exact error in the iOS testing when this is accessible?

Reply:
For whatever reason, having that view be accessible in iOS makes finding the header with testID='Base_TestPage' always fail. 

The problem is that the view also has to be accessible because it'll get optimized out in win32 if it isn't. I'm pretty sure this is a bug because the [react native docs](https://reactnative.dev/docs/view#testid) say that including a testID prop disables layout optimization.

Reply:
If we do have to make this distinction for mobile, I would create another boolean value to determine whether or not the view should be accessible rather than reuse `enableSinglePaneView` which is used for another purpose.

Reply:
Changed it to explicitly test whether the current platform is iOS.

Reply:
I'm not sure... I guess our E2E tests (which use xc / xcii tests under the hood) are very tied to the accessibility implementation of React Native Core, so it.. kinda? makes sense that there could be a weird bug like this? Can we file a bug / issue about this so that it's easier to route someone to investigate?

Reply:
Filed a issue, will resolve